statsmodels/tools/eval_measures.py

```diff
@@ -67,6 +68,30 @@ def rmse(x1, x2, axis=0):
     x2 = np.asanyarray(x2)
     return np.sqrt(mse(x1, x2, axis=axis))
 
+def rmspe(y, y_hat, axis=0, zeros=np.nan):
+    """
+    root mean squared percentange error
+    
+    Parameters
+    ----------
+    y, y_hat : array_like
+       The performance measure depends on the difference between these two
+       arrays.
+    axis : int
+       axis along which the summary statistic is calculated
```

Comment:
Capitals here and below as well to start descriptions.

Reply:
And again, hre I'm just following the style of the rest of the script.

Reply:
Done now